.azure-pipelines/official-build-cisco-8000.yml

```diff
@@ -0,0 +1,59 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+schedules:
+- cron: "0 8 * * *"
+  displayName: Daily midnight build
+  branches:
+    include:
+    - master
+    - 202012
+  always: true
+
+trigger: none
+pr: none
+
+resources:
+  repositories:
+  - repository: cisco-8000
+    type: github
+    name: Cisco-8000-sonic/platform-cisco-8000
+    endpoint: cisco-connection
+
+stages:
+- stage: Build
+  pool: sonic
+  variables:
+    CACHE_MODE: wcache
+    SKIP_CHECKOUT: true
+    TERM: ''
+
+  jobs:
+  - template: azure-pipelines-build.yml
+    parameters:
+      buildOptions: 'USERNAME=admin SONIC_BUILD_JOBS=$(nproc) ${{ variables.VERSION_CONTROL_OPTIONS }}'
+      preSteps:
+      - checkout: self
+        submodules: recursive
+        path: s
+        displayName: 'Checkout code'
+      - checkout: cisco-8000
+        submodules: recursive
+        path: s/platform/cisco-8000
+        displayName: 'Checkout cisco-8000'
+      - task: DownloadGitHubRelease@0
+        inputs:
+          connection: cisco-connection
+          userRepository: Cisco-8000-sonic/platform-cisco-8000
+          defaultVersionType: specificTag
+          version: 202012-v0.2
+          itemPattern: 'artifactory-*.tar.gz'
```

Comment:
Can we get the artifactory-*.tar.gz here based on the branch the az-pipeline is being run for ? 
If do we can use the same pipeline.yml file for getting cisco-build done for 202106 branch as well. The artifactiory we need in 202106 would be different.

Reply:
@judyjoseph , I have changed to use the value specified in cisco-8000.ini, do you think it is good now?